warehouse/models/mart/transit_database/_mart_transit_database.yml

```diff
@@ -51,6 +51,19 @@ models:
           metabase.semantic_type: type/FK
       - name: details
         description:	Text description related to the organization
+      - name: reporting_category
+        description: |
+          Categories we want to hold ourselves accountable to:
+          Core: funded by Caltrans/can control or influence somewhat
+          Other Public Transit: (publicly available or managed) and publicly funded transit
+          Other Transit: transit that isn't the above
+      - name: ntp_id
+        description: NTD ID
+      - name: alias
+      - name: gtfs_static_status
+        description: |
+          Computed value from source data: IF(
+          {Complete RT coverage}+{>= 1 complete RT set (1=yes)}=2,"RT OK","RT Incomplete")
```

Comment:
is this computed in an Airtable calculated column/equation? might be worth clarifying

Reply:
Ope yeah I'd actually copied it in the wrong field. Fixed & clarified